tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es5",
+        "target": "ES2019",
```

Comment:
Bumping this to move us at least slightly closer to the present. ES5 is ancient.